.github/workflows/ci.yml

```diff
@@ -23,56 +29,69 @@ jobs:
       - run: cargo test --release integration -p forge
 
```

Comment:
nit: `cargo test --release --bin snforge` above should be removed as it doesn't test anything